workers/unity/Packages/com.improbable.gdk.core/Components/Option.cs

```diff
@@ -34,6 +59,16 @@ public Option(T value)
             this.value = value;
         }
 
+        /// <summary>
+        ///    Attempts to get the value contained within the Option.
+        /// </summary>
+        /// <param name="outValue">
+        ///     When this method returns, contains the value contained within the Option if the Option is non-empty,
```

Comment:
doesn't return the default constructor, but returns a default value

Reply:
or whatever c# term for that is (unsure)

Reply:
Yes you are correct